contrib/standup-db.sh

```diff
@@ -31,6 +31,7 @@ DEX_KEY_SECRET=$(dd if=/dev/random bs=1 count=32 2>/dev/null | base64)
 export DEX_OVERLORD_DB_URL=$DEX_DB_URL
 export DEX_OVERLORD_KEY_SECRETS=$DEX_KEY_SECRET
 export DEX_OVERLORD_KEY_PERIOD=1h
+export DEX_OVERLORD_ADMIN_API_SECRET=$(dd if=/dev/random bs=1 count=128 2>/dev/null | base64)
```

Comment:
There is a method to my madness. I set the `DEX_KEY_SECRET` over by itself because it is shared.

Then for each binary, I set the environment variables close to where the binary is invoked; I think it's more useful there, because it shows in a glance all the things one needs to start up an overlord or worker.